### PR TITLE
Add Query File - TESTED

### PR DESCRIPTION
**Pull request by Liang-Hsuan**
Fix the ACL of `ServiceProviderSeeTPatient `, `ServiceProviderSeePatientInsurancePolicy`, `InsuranceProviderReadOnlyTheirPatient`.

### CODE REVIEW

File: app/lib/logic.js [hunk 1]
```diff
@@ -229,17 +229,14 @@ function checkParticipantAuthorizedToViewMedicalRecord(mr, p) {
  * @return {boolean}
  */
 function insurerInsureeRelated(insurer, insuree) {
```
Comment (Tan-He) [inline app/lib/logic.js]: What if A has an insurance policy with company B and an insurance policy with company C? 
Can B view the policy A signed with C?

Comment (Tan-He) [reply app/lib/logic.js]: > rule ServiceProviderSeePatientInsurancePolicy {
    description: "Grant service providers to see list of their patients insurance policy"
    participant(p): "live.beyondblock.network.ServiceProvider"
    operation: READ
    resource(ip): "live.beyondblock.network.InsurancePolicy"
    condition: (patientServiceProviderRelated(ip.insuree, p))
    action: ALLOW
}

Comment (Liang-Hsuan) [reply app/lib/logic.js]: I don't know about should B see the policy A signed with C, but for the current logic B cannot. Maybe this kind of detailed permission we can deal with it in the future as it is trivial to allow B to see A's other insurance policies if A is B's client.

Comment (Tan-He) [reply app/lib/logic.js]: but from B's perspective, wouldn't patientServiceProviderRelated(ip.insuree, p) also return true for the contract A signed with C in this case?

Comment (Liang-Hsuan) [reply app/lib/logic.js]: First of all `patientServiceProviderRelated(ip.insuree, p)` is used between service provider and patient, not insurance provider and patient.

Second, both `patientServiceProviderRelated` and `insurerInsureeRelated` will only return true only if the `(patient==_$patient) AND (provider==_$provider)` and `(insurer==_$insurer) AND (insuree==_$insuree)` respectively.

Comment (Tan-He) [reply app/lib/logic.js]: my bad. I meant insurerInsureeRelated()

Comment (Liang-Hsuan) [reply app/lib/logic.js]: But the `insurer` won't be B, despite `insuree` is A. The `AND` statement will return `false`.

Comment (Tan-He) [reply app/lib/logic.js]: Nvm. I was confused by the ACL rule names. I thought patientServiceProviderRelated function was called by the InsuranceProviderSeeInsurancePolicy.

**Pull request merged**